superbench/benchmarks/micro_benchmarks/ib_loopback_performance.py

```diff
@@ -181,13 +180,11 @@ def _process_raw_result(self, cmd_idx, raw_output):
 
         valid = False
         content = raw_output.splitlines()
-        try:
-            metric_set = set()
-            for line in content:
+
+        metric_set = set()
+        for line in content:
+            try:
                 values = list(filter(None, line.split(' ')))
-                # Filter useless line
-                if len(values) != 5 or not re.match(r'\d+', values[0]) or not re.match(r'\d+.\d+', values[-2]):
```

Comment:
keep the `len(values)` check?

Reply:
added, but it can not filter all useless line in fact